deploy/charts/trust-manager/templates/webhook.yaml

```diff
@@ -1,3 +1,47 @@
+{{- /*
+$ca is always generated here even if issueIsolatedCert is false because we need it to be
+visible in the scope for the ValidatingWebhookConfiguration below.
+
+genCA has two args - first is cert commonName and second is validity in days (9125 = ~25 years)
+
+We don't write this CA to a secret because we don't want it to be used for any other purpose!
```

Comment:
This statement seems untrue. The CA is written to a secret below, or am I misunderstanding something?

Reply:
I'll improve the comment - it should say that the ca's private key isn't written to a secret!